libsolidity/ast/Types.h

```diff
@@ -416,6 +416,8 @@ class RationalNumberType: public Type
 
 private:
 	rational m_value;
+
+	virtual std::tuple<bool, rational> parseRational(std::string const& _value, bool _allowRadixPoint);
```

Comment:
I think `_allowRadixPoint` is not used in the implementation, is it?
Any reason to declare it virtual?